<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="MonikaYuvaraj" failures="0" tests="4" name="api.utilities.DDTest" time="5.556" errors="1" timestamp="2025-07-25T01:44:25 IST" skipped="0">
  <testcase classname="api.utilities.DDTest" name="testPostUser" time="4.213"/>
  <testcase classname="api.utilities.DDTest" name="testPostUser" time="0.486"/>
  <testcase classname="api.utilities.DDTest" name="testPostUser" time="0.857"/>
  <testcase classname="api.utilities.DDTest" name="testDeleteUserById" time="0.000">
    <error message="
Method public void api.utilities.DDTest.testDeleteUserById(java.lang.String) requires a @DataProvider named : UserIDs in class api.utilities.DataProviders" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Method public void api.utilities.DDTest.testDeleteUserById(java.lang.String) requires a @DataProvider named : UserIDs in class api.utilities.DataProviders
at org.testng.internal.Parameters.findDataProvider(Parameters.java:535)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:737)
at org.testng.internal.invokers.ParameterHandler.handleParameters(ParameterHandler.java:72)
at org.testng.internal.invokers.ParameterHandler.createParameters(ParameterHandler.java:51)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:915)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- testDeleteUserById -->
</testsuite> <!-- api.utilities.DDTest -->
